### ISSUE TITLE
Accessibility Issues

### ISSUE THREAD

**Issue by Johnston524** [select2/select2#5676]
Hi,

I had a report done on a form that I'd created for JAWS accessibility and the following items were identified as issues with Select 2 fields. I'm using the latest release with no changes made.

- 'Select an option' do not display the list of items when the ‘Enter’ key is pressed
- form field dropdown label is not announced when users tab through the form field (JAWS 2019 & Chrome)
- ‘Select an option or more than one’ - form field is skipped when the user presses the down arrow key from the label 
- form field doesn’t trap the keyboard focus (JAWS 2019 & IE11). 
- form field stays open when the user navigates away (JAWS 2019 & IE11)
- Pressing the down arrow while the virtual focus is located on the ‘Enter answer here …’ question to a dropdown results in JAWS announcing, ‘main region ends’ (JAWS 2019 & Chrome and JAWS 2019 & IE11)
- ‘Remove’ button is announced as ‘times’ by screen readers (all browsers)

I've trawled through various GitHub/Stackoverflow pages for solutions each with varying degrees of helpfulness.

**Comment by kevin-brown** [comment]
This information and great and very useful for us! We're currently tracking general accessibility issues at https://github.com/select2/select2/issues/3744 and for more specific issues we prefer having them broken down into individual tickets (so we can track and fix them individually).
